### DISCUSSION TITLE
Fillable filter/refiner in PNP Modern Search

### DISCUSSION THREAD

**Discussion by goodescott** [microsoft-search/pnp-modern-search#3868] [Q&A]
Is it possible to have a refiner/filter that is fillable from a managed property that is a text string?  For example I create a column named "Feeling" that is a string.  You can add any text you want.  I map "Feeling" to refinablestring01 and an index happens.  Users add happy, sad, don't know, etc. in free form text, not a choice.  Can I add this property to a filter web part and be able to fill in any text for the search?  Thanks for any help.

**Comment by wobba** [comment]
@goodescott can you try to explain a bit more what you are trying to accomplish in terms of user experience?

**Reply by goodescott** [reply]
Using PNP Modern Search.  Create a page with web parts including a searh box, searh results and search filters.  I want a filter that can be filled in, not just a drop down of choices.  Is that possible?

**Reply by wobba** [reply]
Try to use the combo-box template https://microsoft-search.github.io/pnp-modern-search/usage/search-filters/#configuration

**Reply by goodescott** [reply]
Oh I see.  It looks like teh box is fillable or drop down.  Thanks Wobba.  I will give it a shot.
